diy_asylum_frontend/src/reducers/helpText.js

```diff
@@ -0,0 +1,42 @@
+import { INCREMENT_STEP, DECREMENT_STEP } from "./form";
```

Comment:
I think I see where you're going with this. The idea is to reset the helper form to blank whenever the user goes forwards or back in step forms, right?

That's a good edge case to cover, but I think that mixing actions in the reducers (at this stage) is a little much. Let's either, in the component level, explicitly call each action *or* , if we really do what to mix the actions, import only `setInputHelpText` into the form reducer

since it's just one import instead of two

Reply:
I'll just make the next page buttons fire off two actions, seems innocuous. I tend to want to put more logic in the reducers and less in the view, but I guess the common convention is the opposite.